apps/api/scripts/cert-renewal.sh

```
@@ -0,0 +1,231 @@
+            || log "WARN" "Failed to send notification"
+    fi
+}
+
+# Check certificate expiration
+check_certificate_expiry() {
+    if [[ ! -f "$CERT_PATH/cert.pem" ]]; then
+        log "WARN" "Certificate not found at $CERT_PATH/cert.pem"
+        return 1
+    fi
+    
+    local expiry_date=$(openssl x509 -in "$CERT_PATH/cert.pem" -noout -enddate | cut -d= -f2)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)